hedgehog/test/test.hs

```diff
@@ -1,18 +1,10 @@
-import           Control.Monad (unless)
-import           System.IO (BufferMode(..), hSetBuffering, stdout, stderr)
-import           System.Exit (exitFailure)
+import           Hedgehog (runTestsExitFail)
 
 import qualified Test.Hedgehog.Text
 
 
 main :: IO ()
-main = do
-  hSetBuffering stdout LineBuffering
-  hSetBuffering stderr LineBuffering
```

Comment:
Don't we need this? Should probably be in `runTests`

https://github.com/ambiata/disorder.hs/blob/master/disorder-core/src/Disorder/Core/Main.hs#L26-L28